analyzer/src/funTest/assets/projects/synthetic/maven/pom.xml

```diff
@@ -15,7 +15,7 @@
         </license>
     </licenses>
     <properties>
-        <jenkins-ci.url>http://repo.jenkins-ci.org/releases</jenkins-ci.url>
+        <jenkins-ci.url>https://repo.jenkins-ci.org/releases</jenkins-ci.url>
```

Comment:
It would be useful to add a link to the rule in the commit message to explain why this was reported by LGTM:
https://lgtm.com/rules/1511115648721/

Reply:
Done.